packages/app/src/App.tsx

```diff
@@ -130,7 +130,7 @@ const AppRouter = app.getRouter();
 
 const routes = (
   <FlatRoutes>
-    <Navigate key="/" to="catalog" />
+    <Route path="/" element={<Navigate to="catalog" />} />
```

Comment:
`/` okay here?

Reply:
Yep it's referring to the absolute path "/", as in only the root of the app